src/sentry/tagstore/base.py

```diff
@@ -233,7 +233,9 @@ def get_group_tag_value_qs(self, project_id, group_id, environment_id, key, valu
         """
         raise NotImplementedError
 
-    def get_groups_user_counts(self, project_ids, group_ids, environment_ids, start=None, end=None):
+    def get_groups_user_counts(
+        self, project_ids, group_ids, environment_ids, snuba_filters, start=None, end=None
+    ):
```

Comment:
Adding `snuba_filters` here kind of breaks the tagstore abstraction, since the point is that it is meant to be backend agnostic. To be honest, I'm not sure that we need this to exist in tagstore at all, so I'm ok with this for now. In a follow-up we should look into moving `get_groups_user_count` outside of tagstore.

Reply:
Added this point to [the card](https://getsentry.atlassian.net/browse/WOR-206) blocking the general release.